src/Promitor.Tests.Unit/Validation/Scraper/Metrics/Sinks/StatsDMetricSinkValidationStepTests.cs

```diff
@@ -87,6 +88,69 @@ public void Validate_StatsDWithoutHost_Fails(string host)
             PromitorAssert.ValidationFailed(validationResult);
         }
 
+        [Fact]
+        public void Validate_StatsDWithGenevaFormatWithoutGenevaConfiguration_Fails()
+        {
+            // Arrange
+            var runtimeConfiguration = CreateRuntimeConfiguration();
+            runtimeConfiguration.Value.MetricSinks.Statsd.MetricFormat = StatsdFormatterTypesEnum.Geneva;
+            runtimeConfiguration.Value.MetricSinks.Statsd.Geneva = null;
+
+            // Act
+            var azureAuthenticationValidationStep = new StatsDMetricSinkValidationStep(runtimeConfiguration, NullLogger<StatsDMetricSinkValidationStep>.Instance);
+            var validationResult = azureAuthenticationValidationStep.Run();
+
+            // Assert
+            PromitorAssert.ValidationFailed(validationResult);
+        }
+        [Fact]
+
+        public void Validate_StatsDWithGenevaFormatWithoutGenevaAccountConfiguration_Fails()
```

Comment:
Let's verify against multiple scenarios
```suggestion

        [Theory]
        [InlineData("")]
        [InlineData(" ")]
        [InlineData(null)]
        public void Validate_StatsDWithGenevaFormatWithoutGenevaAccountConfiguration_Fails(string accountName)
```